snapshot/policy/scheduling_policy_test.go

```diff
@@ -10,54 +10,59 @@ import (
 	"github.com/kopia/kopia/snapshot/policy"
 )
 
+//nolint:maintidx
```

Comment:
is this `//nolint` directive still needed?
```suggestion
```

Reply:
yes.  apparently adding 'name' to each section is enough to put us over.  I could split the test up into multiple functions if that is preferred.